### ISSUE TITLE
TypeDoc incorrectly marks properties with more than one source as overridden

### ISSUE THREAD

**Issue by Gerrit0** [TypeStrong/typedoc#1112]
Surely there's a better title... I ran into this while investigating #642, where the `stack` property on a custom `Error` type will show up as overridden since it is defined both in `lib.es5.d.ts` and `globals.d.ts` (from `@types/node`)

- [X] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates

## Expected Behavior

Given the following code, the `stack` property should show up under `B` as inherited from `A`, but not as overridden from `A`.

```ts
interface A {
    stack?: string
}
class A {
    stack?: string
}
class B extends A {}
```

## Actual Behavior

![image](https://user-images.githubusercontent.com/19329837/66710532-cd02ae80-ed37-11e9-9d14-4ca7d894cf89.png)

## Environment
  - Typedoc version: `master`
  - Node.js version: 12.x
  - OS: Windows

**Comment by Gerrit0** [comment]
This should probably be addressed when fixing #1191

**Comment by Gerrit0** [comment]
![image](https://user-images.githubusercontent.com/19329837/103322903-70071d00-49fd-11eb-918e-54cbe56f5c63.png)
This works in 0.20.4
